src/niweb/niweb/templates/base.html

```diff
@@ -142,20 +142,15 @@
                         <li class="divider"></li>
                         <li class="nav-header"><i class="icon-list"></i> Browse types</li>
                         {% type_menu %}
+                        {% comment %}
                         <li class="divider"></li>
                         <li class="nav-header"><i class="icon-briefcase"></i> Reports</li>
-                        <li><a href="/reports/hosts/">Host reports</a></li>

```

Comment:
Instead of removing it (and by that removing comparability with NORDUnets NI) can we not make it configurable using the global_preferences_registry and a template tag as it is done for the type menu.

https://github.com/SUNET/ni/blob/master/src/niweb/apps/noclook/templatetags/noclook_tags.py#L16